app/Model/Course.php

```diff
@@ -158,4 +159,14 @@ public function getAttendanceDeadlineEndAttribute(): ?CarbonImmutable
             ? $this->attendance_deadline->endOfDay()
             : null;
     }
+
+    /**
+     * 受講期限の設定を取得
+     *
+     * @return HasOne<CourseDeadline, $this>
+     */
+    public function deadline(): HasOne
```

Comment:
テーブル名に合わせて、`courseDeadline()`にして欲しいです！
コントローラー側も修正が必要になります！
```suggestion
    public function courseDeadline(): HasOne
```